indico/modules/events/layout/controllers/layout.py

```diff
@@ -74,12 +75,28 @@ def _process(self):
 
 
 class RHLayoutEdit(RHLayoutBase):
-    def _process(self):
+    def _process_request(self):
         if self.event.type_ == EventType.conference:
             return self._process_conference()
         else:
             return self._process_lecture_meeting()
 
+    def _process_GET(self):
+        return self._process_request()
+
+    def _process_POST(self):
+        old_values = layout_settings.get_all(self.event)
+        ret = self._process_request()
+        new_values = layout_settings.get_all(self.event)
+        changes = {name: [old_values[name], value, 'string']
+                   for name, value in new_values.items()
+                   if old_values[name] != value}
+        changes['_diff'] = True
```

Comment:
This breaks when changing the "Name format" since it tried to store an enum member in the DB. Have a look at `_log_event_update` and more important `make_diff_log` - you may want to do something similar here.

That way you'd also get nicer entries in the log message than just the raw names of the settings.

Reply:
It took a bit to figure out but it's now done.